anillo/middlewares/cors.py

```diff
@@ -0,0 +1,23 @@
+import functools
+
+def wrap_cors_response(func=None, *, allow_origin='*', allow_headers='Origin, X-Requested-With, Content-Type, Accept'):
```

Comment:
This line is too large!

Also, the `allow_headers` I think should be a set of lower case strings instead of plain strings. Is much less error prone handling with sets that with string concatenations.